### PR TITLE
Conda on wICE management - interactive session

### PR DESCRIPTION
**Pull request by StefaanVerwimp**
Added short statement in "Conda on wICE" that the interactive Slurm partition can be used for Conda environment management on wICE.

### CODE REVIEW

**Review by stevenvdb — CHANGES_REQUESTED** [review]
The content is ok for me, but to make the link work, you need to put `:ref:` in front of \`interactive Slurm partition on wICE<submit to wice interactive node>\`

**Review by stevenvdb — APPROVED** [review]

File: source/leuven/wice_advanced_guide.rst [hunk 1]
```diff
@@ -145,7 +145,11 @@ Conda on wICE
 
 As the operating system and hardware are different on Genius and wICE, we advise
```
Comment (moravveji) [inline source/leuven/wice_advanced_guide.rst:146]: As of monday 25/9/23, the OS of both machines will be identical. So, please update this line, too.

**Review by moravveji — CHANGES_REQUESTED** [review]

**Comment by moravveji** [comment]
Furthermore, I like to propose an additional change on paragraph one, sentence 3.
This should totally go away:
"Since the operating system (Rocky Linux) differs from the one on Genius (CentOS), a separate compilation is strictly required in most cases anyway. "

**Comment by StefaanVerwimp** [comment]
The parts mentioning a difference in operating systems between Genius and wICE have been removed from wice_advanced_guide.rst.

**Comment by moravveji** [comment]
@StefaanVerwimp : Hey Stefaan. Can you please take another look at my comments and see if we can round off this PR?
Best is that such PRs would not last long. Thanks.

File: source/leuven/wice_advanced_guide.rst [hunk 2]
```diff
@@ -12,9 +12,7 @@ Compiling software
 The wICE compute nodes feature Intel processors nicknamed IceLake, which are
 the successors of the SkyLake and CascadeLake architectures which you can find
 on Genius. Although architectural differences are rather small, it is highly
-recommended to compile a version of your code specifically for wICE. Since the
-operating system (Rocky Linux) differs from the one on Genius (CentOS), a
-separate compilation is strictly required in most cases anyway. A good approach
+recommended to compile a version of your code specifically for wICE. A good approach
```
[the remark this answers is not in the document]
Comment (moravveji) [reply source/leuven/wice_advanced_guide.rst:15]: @StefaanVerwimp : If you could make one last fix, we can have this branch merged.
Is it possible to revisit your PR?
Thanks

**Review by moravveji — CHANGES_REQUESTED** [review]

**Review by moravveji — APPROVED** [review]
I approve, and will take over afterwards for the last correction.

**Pull request merged**